aad-pod-identity-rbac.yml

```diff
@@ -0,0 +1,257 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: aad-pod-id-nmi-service-account
+  namespace: default
```

Comment:
Please use this [terraform resource](https://www.terraform.io/docs/providers/kubernetes/r/service_account.html)